community/self-hosting-novu/introduction.mdx

```diff
@@ -0,0 +1,21 @@
+---
+title: 'Introduction'
+description: ''
+icon: 'code'
+---
+
+Self-hosting Novu involves setting up and maintaining your communication infrastructure using your own servers.
+While this offers full control and customization,
+some features which are available only for Novu’s cloud-managed won't be available in a self-hosted environment.
```

Comment:
Is this line break necessary? If not, please remove.

Reply:
Sure, I used the old page. Let me rewrite this